evo-delivery-app/evo-delivery-backend/src/types/driver.type.ts

```diff
@@ -0,0 +1,6 @@
+export type Driver = {
+  id: string;
```

Comment:
Note that when inserting a new driver the id is not known, so I think it should be Optional